Gandalan.IDAS.Contracts/Token/JwtTokenService.cs

```diff
@@ -74,15 +76,19 @@ public string GenerateToken(UserAuthTokenDTO authToken, RefreshTokenDTO refreshT
             var credentials = GetSigningCredentials(privateKey);
 
             // roles
+            // roles & rights
+            var roleCodes = new List<string>();
             var rightCodes = new List<string>();
             if (authToken.Benutzer?.Rollen != null)
             {
+                roleCodes = authToken.Benutzer.Rollen.Select(r => r.Name).ToList();
                 rightCodes = authToken.Benutzer.Rollen.SelectMany(r => r.Berechtigungen.Select(b => b.Code)).Distinct().ToList();
             }
 
             // claims
             var rights = new List<Claim>();
```

Comment:
Rename to `claims` or something familiar

Reply:
`claims` is too generic IMO, our structure is internally `roles` and `right` codes, so I'd stick to that